fetch/worker.go

```diff
@@ -161,6 +161,12 @@ func getTxResultFromBatch(blocks []*types.Block, client Client) ([][]*types.TxRe
 		return getTxResultsSequentially(blocks, client)
 	}
 
+	indexOfBlockHeight := make(map[int64]int)
```

Comment:
You can init this map with a pre-existing size `make(map[int64]int, len(blocks))`

Reply:
Fixed in this commit. 
https://github.com/gnolang/tx-indexer/pull/52/commits/760e6c75c092948fc272b24af7637df7baed28f5